yoti-sdk-api/src/main/java/com/yoti/api/client/spi/remote/call/SignedRequestBuilderFactory.java

```diff
@@ -23,7 +25,7 @@
     }
 
     public SignedRequestBuilder create() {
-        return new SignedRequestBuilder(pathFactory, signedMessageFactory, headersFactory, jsonResourceFetcher, rawResourceFetcher, imageResourceFetcher);
+        return new SignedRequestBuilder(pathFactory, signedMessageFactory, headersFactory, jsonResourceFetcher, rawResourceFetcher, imageResourceFetcher, MultipartEntityBuilder.create());
```

Comment:
If we need a new MultipartEntityBuilder every time, just do it in the constructor for SignedRequestBuilder.

Do we actually need to create it upfront...?

Reply:
Makes it much easier to unit test